src/rocket_component.py

```diff
@@ -41,50 +48,52 @@ def __init__(self, position: Position) -> None:
   def calc_mass(self, time_step) -> float:
     """
     calculate total mass of the system
-    :return float: mass of the rocket at a point in time
+    :param float time_step: time difference from one state to the next
+    :return float mass: mass of the rocket at a point in time
     """
 
     mass = self.mass_structure + self.mass_fuel - (self.fuel_flow_rate * time_step)
     return mass
 
 
-  def calc_drag_force(self) -> float:
+  # Calculate Forces
+  def calc_drag_force(self) -> Vector:
     """
     calculate the drag force acting on the rocket
     :return float: drag
     """
 
     drag_coefficient = 0.75  # approximation, replace later
     drag_area = math.pi * (self.width / 2)**2
-    drag_force = drag_coefficient * self.atmosphere.density * drag_area * self.velocity ** 2 / 2
-    return drag_force
+    drag = drag_coefficient * self.atmosphere.density * drag_area * self.velocity ** 2 / 2
```

Comment:
What we're returning is our drag, a Vector object, but this value here is the drag_magnitude.  I would say we want to name it to be more specific about what it is.  You'll need edits where it's then used too
```suggestion
    drag_magnitude = drag_coefficient * self.atmosphere.density * drag_area * self.velocity ** 2 / 2
```